pkgs/applications/virtualization/colima/default.nix

```diff
@@ -1,54 +1,66 @@
 { lib
-, buildGo118Module
+, buildGoModule
 , fetchFromGitHub
 , installShellFiles
 , lima
 , makeWrapper
+, qemu
+, testers
+, colima
+, runCommand
```

Comment:
Is this used?

Reply:
looks more like forgotten debug lines

Reply:
@06kellyjac
runCommand slipped through, thanks for catching that.

- qemu is used by the wrapper
- testers and colima (the package itself) are used in passthru for testing. Adopted from here https://nixos.org/manual/nixpkgs/stable/#chap-testers

Reply:
yeah the comment was just for the runcommand line, not sure why it grabbed the whole section